### PR TITLE
Coding style

### LINKED ISSUE #68: Improve documentation in app_common

### PR DESCRIPTION
**Pull request by IsaacYangSLA**
Fix #68 

pydocstyle with google convention and add_ignore D100,D101,D102,D103,D104,D105,D106 on the following folders
app_common
fuel/[common,sec]
lighter
security

### CODE REVIEW

File: runtest.sh [hunk 1]
```diff
@@ -21,10 +21,12 @@ function print_style_fail_msg() {
     echo "Please run ${green}./runtest.sh${noColor} to check errors and fix them." 
 } 
 
-grep -V
-echo "set to +e"
+# grep -V
+# echo "set to +e"
```
Comment (YuanTingHsieh) [inline runtest.sh:25]: I think we can clean these up later.

File: nvflare/fuel/common/ctx.py [hunk 2]
```diff
@@ -41,11 +38,8 @@ def clear_props(self):
 
 
 class BaseContext(SimpleContext):
-    """
-    A SimpleContext with threading locks.
-    """
-
     def __init__(self):
+        """A SimpleContext with threading locks."""
```
Comment (YuanTingHsieh) [inline nvflare/fuel/common/ctx.py:42]: we might want to improve this.
What does this context serve? Why it needs to have asked_to_stop parameter

File: nvflare/app_common/aggregators/intime_accumulate_model_aggregator.py [hunk 3]
```diff
@@ -24,7 +24,8 @@
 
 class InTimeAccumulateWeightedAggregator(Aggregator):
     def __init__(self, exclude_vars=None, aggregation_weights=None, expected_data_kind=DataKind.WEIGHT_DIFF):
-        """Perform accumulated weighted aggregation
+        """Perform accumulated weighted aggregation.
+
```
Comment (YuanTingHsieh) [inline nvflare/app_common/aggregators/intime_accumulate_model_aggregator.py:28]: not sure if these changes going to conflict with #165

File: runtest.sh (hunk 1 above)
Comment (IsaacYangSLA) [reply runtest.sh:25]: Agree.  This script needs some cleanup after we have a stable CI pipeline.

File: nvflare/fuel/common/ctx.py (hunk 2 above)
Comment (IsaacYangSLA) [reply nvflare/fuel/common/ctx.py:42]: This context is only used in low level library and actually ask_to_stop() is never called.  We can remove this parameter.

File: nvflare/app_common/aggregators/intime_accumulate_model_aggregator.py (hunk 3 above)
Comment (IsaacYangSLA) [reply nvflare/app_common/aggregators/intime_accumulate_model_aggregator.py:28]: This class should behave the same after #165 is merged.  That PR does not change the API of this class.  However, there are some sentences been rephrased.  I can resolve the conflicts if this one is merged after #165.

**Review by nvkevlu — APPROVED** [review]

**Review by nvkevlu — APPROVED** [review]

**Review by YuanTingHsieh — APPROVED** [review]
LGTM.

**Pull request merged**